server/controllers/pagenation/index.js

```diff
@@ -0,0 +1,33 @@
+const { Work, User } = require('../../database/models');
+const { customError } = require('../errors');
+const { workIdValidation } = require('../../utils/validation');
+
+const getPageData = async (req, res, next) => {
+  try {
+    const { providerId } = req.params;
+    const { page } = req.query;
+    const objectToValidation = { userId: providerId, pageNum: page };
+    const { userId, pageNum } = await workIdValidation.validateAsync(objectToValidation);
+    const user = await User.findByPk(userId);
+    if (!user) {
+      throw customError('User not found', 404);
+    }
+    const pageData = await Work.findAndCountAll({
+      limit: 4,
+      offset: (pageNum - 1) * 4,
+      where: { userId }, // conditions
+    });
+    console.log(pageData);
+    if (!pageData.count) {
```

Comment:
If there is no works; response status code should be 400 and message  "There Is No Works".
I think 'Page Not Found' not descriptive message